cmd/deletereleasingpolicy.go

```diff
@@ -0,0 +1,80 @@
+// Copyright © 2019 Developer <developer@vamp.io>
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+package cmd
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/magneticio/forklift/core"
+	"github.com/magneticio/forklift/logging"
+	"github.com/magneticio/forklift/util"
+	"github.com/spf13/cobra"
+)
+
+var deleteReleasingpolicyCmd = &cobra.Command{
+	Use:   "releasingpolicy",
+	Short: "Delete a releasingpolicy",
+	Long: AddAppName(`Delete a new releasingpolicy
+    Example:
+    $AppName delete releasingpolicy name --organization org --environment env --file ./releasingpolicydefinition.json -i json`),
```

Comment:
I don't think the file parameter is really needed, am I right?

Reply:
You are right. Fixing it. It was a copy-paste error.